cloud/services/virtualmachines/virtualmachines_test.go

```diff
@@ -1507,107 +1507,3 @@ func TestReconcileVM(t *testing.T) {
 		})
 	}
 }
-
-func TestDeleteVM(t *testing.T) {
```

Comment:
we actually want to keep this test, it's testing https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/a6d3f30ebf16891446ca0104926cc41516905b12/cloud/services/virtualmachines/virtualmachines.go#L216, not the same Delete

my bad, did not notice this when reviewing earlier

Reply:
Ah, let me return it back.